contracts/test/GrantRound.test.ts

```diff
@@ -23,21 +25,21 @@ describe('GrantRound', function () {
     mpUser: SignerWithAddress; // matching pool user
   let registry: GrantRegistry;
   let roundContract: Contract;
-  let mockERC20: MockContract;
-  const balances: string[] = ['100', '200'];
+  let mockERC20: MockToken;
+  const balances: string[] = ['100', '2000'];
```

Comment:
If I'm understanding this right, this is just an array to capture constants for expected balances throughout the test suite. Do I have that right @corydickson? If so, we ought to pull them into two separate named constants. It's definitely a bit annoying, but I think it will make reading the tests easier at the callsite. Right now, seeing `balances[0]` or `balances[1]` seems cryptic, and would read better if it was something named liked `DONOR_AMOUNT` or `DonorAmount` or whatever (I don't feel strongly about the casing convention).